### PR TITLE
Drawing Classifier: Activated the pretrained model

### PR DESCRIPTION
**Pull request by shantanuchhabra**
Activated the pretrained model as default for training, and added unit tests as well.

### CODE REVIEW

File: src/unity/python/turicreate/toolkits/_pre_trained_models.py [hunk 1]
```diff
@@ -244,11 +244,13 @@ def get_model_path(self, format):
...
 
 class DrawingClassifierPreTrainedModel(object):
-    def __init__(self, model_url = None):
+    def __init__(self, model_url = 'auto'):
         filename = "drawing_classifier_pre_trained_model_245_classes_v0.params"
         self.source_url = (_urlparse.urljoin(MODELS_URL_ROOT, filename) 
-            if model_url is None 
+            if model_url == 'auto' 
             else model_url)
+        # Think about how to bypass the md5 checksum if the user wants
```
Comment (TobyRoseman) [inline src/unity/python/turicreate/toolkits/_pre_trained_models.py]: Should this be a "TODO" comment?

File: src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py [hunk 2]
```diff
@@ -85,7 +85,9 @@ def create(input_dataset, target, feature=None, validation_set='auto',
 
     pretrained_model_url : string optional
         A URL to the pretrained model that must be used for a warm start before
-        training.
+        training. By default, we load the pretrained model we trained and uploaded
+        on https://docs-assets.developer.apple.com/turicreate/models/
```
Comment (TobyRoseman) [inline src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py]: I wouldn't mention the URL or that we trained it. I would just say "By default, we load the pretrained model."

**Comment by srikris** [comment]
Please consider #1614

File: src/unity/python/turicreate/toolkits/_pre_trained_models.py (hunk 1 above)
Comment (shantanuchhabra) [reply src/unity/python/turicreate/toolkits/_pre_trained_models.py]: Done.

File: src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py (hunk 2 above)
Comment (shantanuchhabra) [reply src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py]: Fixed.

**Comment by nickjong** [comment]
Please do not add PRs to a milestone when they are already represented by an Issue in that milestone. We don't need to double count work.

**Comment by shantanuchhabra** [comment]
> Please do not add PRs to a milestone when they are already represented by an Issue in that milestone. We don't need to double count work.

Sorry about that. I noticed the Milestone had some closed PRs, and also wanted to track the 5.4 PRs against the >5.4 PRs for reviewers' convenience.

**Comment by shantanuchhabra** [comment]
Fixes #1614

**Comment by srikris** [comment]
LGTM. I assume this is for 5.4. If so, let's merge and test

**Pull request merged**